components/advocates/MeetTheAdvocates.vue

```diff
@@ -17,26 +17,30 @@
         </p>
       </div>
     </div>
-    <AppFiltersResultsLayout class="meet-the-advocates__filters-result-section">
+    <UiAppFiltersResultsLayout
+      class="meet-the-advocates__filters-result-section"
+    >
       <template #filters-on-m-l-screen>
-        <AppFieldset :label="filter.label">
+        <UiAppFieldset :label="filter.label">
           <client-only>
-            <cv-checkbox
+            <bx-checkbox
               v-for="option in filter.options"
               :key="option"
               :checked="isRegionFilterChecked(option)"
-              :label="option"
+              :label-text="option"
               :value="option"
-              @change="updateRegionFilter(option, $event)"
+              @bx-checkbox-changed="
+                updateRegionFilter(option, $event.target.checked)
+              "
             />
           </client-only>
-        </AppFieldset>
+        </UiAppFieldset>
       </template>
       <template #filters-on-s-screen>
-        <AppMultiSelect
+        <UiAppMultiSelect
           :label="filter.label"
           :options="filter.options"
-          :value="regionFilters"
+          :value="regionFiltersAsString"
```

Comment:
I don't see why are you changing this value to have an `string` instead of an `string[]`, could you give me more info, please?

Reply:
In the **UiAppMultiSelect** we're now using Carbon's Multi Select web component, which needs the values as a comma separated string (https://web-components.carbondesignsystem.com/?path=/docs/components-multi-select--default#bx-multi-select-attributes-properties-and-events).

Before, we used the Carbon Vue component version, which did accept arrays. Hence the change.